runtime/moonbase/src/lib.rs

```diff
@@ -767,6 +767,28 @@ impl pallet_proxy::Config for Runtime {
 	type AnnouncementDepositFactor = AnnouncementDepositFactor;
 }
 
+/// Call filter used during Phase 3 of the Moonriver rollout
+pub struct MaintenanceFilter;
+impl Filter<Call> for MaintenanceFilter {
+	fn filter(c: &Call) -> bool {
+		match c {
+			Call::Balances(_) => false,
+			Call::CrowdloanRewards(_) => false,
+			Call::Ethereum(_) => false,
+			Call::EVM(_) => false,
+			_ => true,
+		}
+	}
+}
```

Comment:
staking is not included on purpose right?

I wonder if we should consider some stupid simple authoring logic during maintenance mode. Like maybe just some hard-coded `Vec<AccountId>` can author during maintenance mode and they get no rewards. IDK I guess this idea exceeds the scope of this PR.